beacon-chain/sync/subscriber_light_client.go

```diff
@@ -0,0 +1,44 @@
+package sync
+
+import (
+	"context"
+
+	"github.com/OffchainLabs/prysm/v6/beacon-chain/core/feed"
+	statefeed "github.com/OffchainLabs/prysm/v6/beacon-chain/core/feed/state"
+	lightclientTypes "github.com/OffchainLabs/prysm/v6/consensus-types/light-client"
+	"google.golang.org/protobuf/proto"
+)
+
+func (s *Service) lightClientOptimisticUpdateSubscriber(_ context.Context, msg proto.Message) error {
+	update, err := lightclientTypes.NewWrappedOptimisticUpdate(msg)
+	if err != nil {
+		return err
+	}
+
+	log.Debug("Saving newly received light client optimistic update. Attested slot %d, Signature slot %d", update.AttestedHeader().Beacon().Slot, update.SignatureSlot())
```

Comment:
This isn't how we usually display information about any object in Prysm. For an example of how its done using the logging library:

https://github.com/OffchainLabs/prysm/blob/develop/beacon-chain/blockchain/head.go#L121

Should do it the same way here and in other places in the PR.